CHANGELOG.md

```diff
@@ -5,6 +5,26 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+## [0.7.0] - 2021-10-29
```

Comment:
Please remove this line (it will be auto-generated when we publish a new release)